es-extension-examples/ext-example-vite/vite.config.js

```diff
@@ -0,0 +1,45 @@
+import reactRefresh from '@vitejs/plugin-react';
+import { defineConfig } from 'vite';
+import dotenv from "dotenv";
+dotenv.config();
+
+const urlPath = process.env.URLPATH || ''
+
+export default defineConfig({
+  /* when in lib mode, must define process.env */
+  define: {
+    'process.env': JSON.stringify({
```

Comment:
When you use Vite in 'lib' mode, process.env is not available unless explicitly shimmed or polyfilled